### PR TITLE
The art of drunken captaining

### PR DESCRIPTION
**Pull request by Code-Cygnet**
Drinking on the job is good.

## About The Pull Request

This PR is basically an attempt at adding a special drink for captains called "Commander and Chief".

## Why It's Good For The Game

Some departments have a special drink that improves the performance of those within them. Life on the station could get rough for a Captain, who oversees the entire station. When times get hard enough that he/she/they might need a drink, they could order a Commander and Chief from the bar before rolling up her/his/their sleeves to put in some work. 

## Changelog
:cl:
add: Added new things - Mind trait, alcohol reagent, chemical reagent, drink sprite and recipe.
imageadd: added commander_and_chief sprite to drinks.dmi
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
this+bicard+salacid+styptic+tricord+milk on a plasmaman is 13.5 brute/tick under 50 hp, fun fact

also, in your cl, delete every line which you've written n/a or such, the bot will add a tag if it sees a cl change. for example, 
add: Added new things - Mind trait, alcohol reagent, chemical reagent, drink sprite and recipe.
add: Added more things
del: n/a
tweak: n/a
balance: n/a
fix: n/a
soundadd: n/a
sounddel: n/a
imageadd: added commander_and_chief sprite to drinks.dmi
imagedel: n/a
spellcheck: n/a
code: added code, basically copy&paste with some edits for drink
refactor: n/a
config: n/a
admin: did NOT mess with admin stuff
server: n/a

would become

add: Metabolism trait for Captain, a new alcoholic drink, Commander in Chief, the drink sprite and recipe.
imageadd: added commander_and_chief sprite to drinks.dmi

**Comment by Code-Cygnet** [comment]
> this+bicard+salacid+styptic+tricord+milk on a plasmaman is 13.5 brute/tick under 50 hp, fun fact
> 
> also, in your cl, delete every line which you've written n/a or such, the bot will add a tag if it sees a cl change. for example,
> add: Added new things - Mind trait, alcohol reagent, chemical reagent, drink sprite and recipe.
> add: Added more things
> del: n/a
> tweak: n/a
> balance: n/a
> fix: n/a
> soundadd: n/a
> sounddel: n/a
> imageadd: added commander_and_chief sprite to drinks.dmi
> imagedel: n/a
> spellcheck: n/a
> code: added code, basically copy&paste with some edits for drink
> refactor: n/a
> config: n/a
> admin: did NOT mess with admin stuff
> server: n/a
> 
> would become
> 
> add: Metabolism trait for Captain, a new alcoholic drink, Commander in Chief, the drink sprite and recipe.
> imageadd: added commander_and_chief sprite to drinks.dmi


OOOOH HECK.. THANKS :D!

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2122,6 +2122,29 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	color = "#ffffc9"
+	boozepwr = 50
+	quality = DRINK_FANTASTIC
+	taste_description = "Tastes like...duty and responsibility?"
+	glass_icon_state = "commander_and_chief"
+	glass_name = "Commander and Chief"
+	glass_desc = "The gems of this majestic chalice represent the departments and their Heads."
+	
+/datum/reagent/consumable/ethanol/commander_and_chief/on_mob_life(mob/living/carbon/M)
+	if(M.mind && HAS_TRAIT(M.mind, TRAIT_CAPTAIN_METABOLISM)))
```
Comment (Sishen1542) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	if(M.mind && HAS_TRAIT(M.mind, TRAIT_CAPTAIN_METABOLISM))
```

Should fix your compile issues.

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 2]
```diff
@@ -794,4 +794,11 @@
...
 	results = list("red_queen" = 10)
-	required_reagents = list("tea" = 6, "mercury" = 2, "blackpepper" = 1, "growthserum" = 1)
\ No newline at end of file
+	required_reagents = list("tea" = 6, "mercury" = 2, "blackpepper" = 1, "growthserum" = 1)
+	
+/datum/chemical_reaction/commander_and_chief
+	name = "Commander and Chief" 
+	id = "commander_and_chief" 
+	results = list("commander_and_chief" = 50) 
+	required_reagents = list("allies_cocktail" = 50, "champagne" = 20, "doctors_delight" = 10, "quintiple_sec" = 10, "screwdriver" = 10)
```
Comment (Sishen1542) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]:
```suggestion
	required_reagents = list("alliescocktail" = 50, "champagne" = 20, "doctorsdelight" = 10, "quintuple_sec" = 10, "screwdrivercocktail" = 10)
more fixies

Comment (Code-Cygnet) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm]: Heck the only one I had right was "champagne" LOL. Thanks for the help :D.

Comment (Sishen1542) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm]: no problemo, gotta start somewhere :heart:

**Comment by Thalpy** [comment]
Can't this be used as a ling check?

**Comment by Code-Cygnet** [comment]
> Can't this be used as a ling check?

Oh.. WELL I imagine it could! A fake captain can't get the benefits of this beverage. Clever question.

**Review by nik707 — APPROVED** [review]

**Pull request merged**